cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.batch.curator.openshift.io/batch.curator.openshift.io_reports.yaml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
```

Comment:
looks like it expects this to be a string and not null.

https://github.com/CCI-MOC/moc-apps/runs/5131654823?check_suite_focus=true#step:4:21

is the `creationTimestamp` even required here?

Reply:
Yes, It should be good without timestamp too. 
@naved001 updated.